Assets/HoloToolkit/Input/Scripts/BasicCursor.cs

```diff
@@ -48,7 +44,25 @@ protected virtual void Awake()
             meshRenderer.enabled = false;
 
             // Cache the cursor default rotation so the cursor can be rotated with respect to the original orientation.
-            cursorDefaultRotation = this.gameObject.transform.rotation;
+            cursorDefaultRotation = gameObject.transform.rotation;
+        }
+
+        private bool GetGazeManagerReference()
+        {
+            gazeManagerReference = GazeManager.Instance;
+
+            if (gazeManagerReference == null)
+            {
+                return false;
```

Comment:
Haha, I do get lucky with an idea occasionally! 😜 Thanks for making this change!